apps/condo/domains/resident/schema/FindOrganizationsByAddressService.js

```diff
@@ -37,85 +38,66 @@ const FindOrganizationsByAddressService = new GQLCustomSchema('FindOrganizations
         },
         {
             access: true,
-            type: 'type FindOrganizationByAddressOrganizationType { id: ID!, name: String!, tin: String!, type: OrganizationTypeType! }',
+            type: 'type FindOrganizationByAddressReceiptType { number: String, category: ID!, balance: String, routingNumber: String, bankAccount: String }',
```

Comment:
What is number?

Reply:
accountNumber / printableNumber. What is it? Make naming more transparent

Reply:
I also need address information here:
<img width="267" alt="image" src="https://github.com/user-attachments/assets/80518964-a372-489b-90a2-d7cd9f88e6c2" />